docs/index.md

```diff
@@ -225,9 +225,13 @@ and it'll end up as structured data in our platform ready to be queried.
 For example, using data from the `User` model above, we could list users from the USA:
 
 ```sql
-SELECT attributes->'result'->>'name' as name, age(attributes->'result'->>'dob') as age
+SELECT
```

Comment:
when I ran this query, I got this error:
<img width="829" alt="image" src="https://github.com/user-attachments/assets/546bf10f-247c-43da-835c-141c501a6032">

So I updated with a query that does work

Reply:
I personally significantly prefer the original example. I suggest:

```sql
SELECT attributes->'result'->>'name' as name, extract(year from (attributes->'result'->>'dob')::date) as "birth year"
FROM records
WHERE attributes->'result'->>'country_code' = 'USA'
```

to demonstrate some fancy-looking SQL. Calculating the age would be better, but apparently datafusion doesn't support extracting years from durations.

Reply:
updated